libraries/rush-lib/src/cli/actions/VersionAction.ts

```diff
@@ -216,7 +216,7 @@ export class VersionAction extends BaseRushAction {
     // Validate result of all subspaces
     for (const subspace of rushConfig.subspaces) {
       // Respect the `ensureConsistentVersions` field in rush.json
-      if (!subspace.shouldEnsureConsistentVersions) {
+      if (!subspace.shouldEnsureConsistentVersions(variant)) {
```

Comment:
The `return;` statement is also incorrect, and should be `continue;` because the `ensureConsistentVersions` configuration is per-subspace.